src/poc/miner_poc_report_handler.erl

```diff
@@ -0,0 +1,106 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% == Miner POC Stream Handler ==
+%% used to relay a receipt or witness report received by a validator
+%% onto the actual challenger
```

Comment:
I thought these got delivered directly, is this still needed?

Reply:
not requried.  Will remove

Reply:
https://github.com/helium/miner/pull/1068/commits/32247902ed59dcdcf354746ce707c1a1aa4c5eda